odos-ui/src/App.tsx

```diff
@@ -49,7 +49,7 @@ function App() {
                     path="visualize"
                     element={
                       <PrivateRoute>
-                        <Visualization />
+                        <Visualization width='400' height='300' />
```

Comment:
It didn't look like these props are used in the code itself. Can we remove this or are you expecting to pass in the size later?

Reply:
If expectation is to use these later for sizing the d3 container, then we would need to add this to the /visualization/index.tsx to pass Typescript validations:
```
type VisualizationProps = {
  width: string;
  height: string;
}

export const Visualization = ({width, height}: VisualizationProps) => {
```